x/evidence/simulation/genesis.go

```diff
@@ -8,23 +8,23 @@ import (
 
 	"github.com/cosmos/cosmos-sdk/codec"
 
+	codectypes "github.com/cosmos/cosmos-sdk/codec/types"
 	"github.com/cosmos/cosmos-sdk/types/module"
 	simtypes "github.com/cosmos/cosmos-sdk/types/simulation"
-	"github.com/cosmos/cosmos-sdk/x/evidence/exported"
 	"github.com/cosmos/cosmos-sdk/x/evidence/types"
 )
 
 // Simulation parameter constants
 const evidence = "evidence"
 
 // GenEvidences returns an empty slice of evidences.
-func GenEvidences(_ *rand.Rand, _ []simtypes.Account) []exported.Evidence {
-	return []exported.Evidence{}
+func GenEvidences(_ *rand.Rand, _ []simtypes.Account) []*codectypes.Any {
```

Comment:
Why was the return type changed here? Can we not return the interface? Does the caller have to convert or do anything?